extensions/wikia/AffiliateService/styles/affiliate-unit.scss

```diff
@@ -0,0 +1,56 @@
+@import "skins/shared/color";
+
+.affiliate-unit {
+    &__wrapper {
+        background-color: rgba($color-page, 0.25);
+        border: solid 1px rgba($color-page-opposite, 0.5);
+        border-radius: 3px;
+        display: flex;
+        height: 100px;
+        margin: 30px 18px;
+
+        @if $is-dark-wiki {
+            border: solid 1px rgba($color-page, 0.5);
+        }
+    }
+
+    &__image-wrapper {
+        background-repeat: no-repeat;
+        background-size: cover;
+        clip-path: polygon(0% 0%,100% 0%,80% 100%,0 100%);
```

Comment:
It's not even supported in edge https://caniuse.com/#feat=css-clip-path :(

Reply:
Was the browser fallback been accepted by Design?

Reply:
No. Only tested in Chrome so far. Doesn't work on Safari either. Will look for another method.